pkg/network/kuryr.go

```diff
@@ -113,7 +113,13 @@ func renderKuryr(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.BootstrapR
 	data.Data["WebhookKey"] = b.WebhookKey
 
 	// Nodes Network MTU
-	data.Data["NodesNetworkMTU"] = b.NodesNetworkMTU
+	mtu  := uint32(b.NodesNetworkMTU)
```

Comment:
Please just change type of `KuryrBootstrapResults.NodesNetworkMTU` to uint32, it has more sense for sure.